v3/integration/genTestCerts/genTestCerts.go

```diff
@@ -0,0 +1,314 @@
+package main
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"os/exec"
+	"strings"
+	"sync"
+	"time"
+)
+
+// Generates a CA, an intermediate, and a leaf certificate and prints their
+// OpenSSL textual output to stdout.
+func main() {
+	ca, err := NewTrustAnchor()
+	if err != nil {
+		panic(err)
+	}
+	fmted, err := OpenSSLFormatCertificate(ca)
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("-------------Trust Anchor-------------")
```

Comment:
Another ultra tiny nit: I would be tempted to make a `headerDelim :=  "-------------"` var and then use a style like `fmt.Printf("%sTrust Anchor%s", headerDelim, headerDelim)` throughout. Mostly so we don't ever have to worry about mismatching the number of `-'s` in a header.